### DISCUSSION TITLE
Got Yarn and NPM installed? Remove cache!

### DISCUSSION THREAD

**Discussion by giscus** [SadmanYasar/sadman-yasar-blogs#14] [Announcements]
# Got Yarn and NPM installed? Remove cache!

Sadman Yasar Sayem personal website using Next.js

https://sadman-yasar-sayem-blogs.vercel.app/posts/remove-disk-space-on-c-drive-windows

**Comment by buudi** [comment]
Thanks! I wasn't aware of this
